lazyflow/classifiers/parallelVigraRfLazyflowClassifier.py

```diff
@@ -14,11 +16,12 @@
 import logging
 logger = logging.getLogger(__name__)
 
+
 class ParallelVigraRfLazyflowClassifierFactory(LazyflowVectorwiseClassifierFactoryABC):
     VERSION = 1 # This is used to determine compatibility of pickled classifier factories.
                 # You must bump this if any instance members are added/removed/renamed.
     
```

Comment:
You'll have to increment this `VERSION` constant.  (I forgot to consider this previously.)

Background: `ClassifierFactory` instances are saved to the project file using the python `pickle` module.  If you add or remove instance members, then the old pickle representation is out-of-sync with the new class.  Bump the version so ilastik knows not to re-use saved classifier factory instances from old project files.